crates/ide_completion/src/render/struct_literal.rs

```diff
@@ -25,29 +27,34 @@ pub(crate) fn render_struct_literal(
 
     let name = local_name.unwrap_or_else(|| strukt.name(ctx.db())).to_smol_str();
 
-    let literal = render_literal(&ctx, path, &name, strukt.kind(ctx.db()), &visible_fields)?;
+    let rendered = render_literal(&ctx, path, &name, strukt.kind(ctx.db()), &visible_fields)?;
 
-    Some(build_completion(ctx, name, literal, strukt))
+    Some(build_completion(&ctx, name, rendered, strukt.kind(ctx.db()), strukt))
 }
 
 fn build_completion(
-    ctx: RenderContext<'_>,
+    ctx: &RenderContext<'_>,
     name: SmolStr,
-    literal: String,
+    rendered: RenderedCompound,
+    kind: StructKind,
     def: impl HasAttrs + Copy,
 ) -> CompletionItem {
     let mut item = CompletionItem::new(
         CompletionItemKind::Snippet,
         ctx.source_range(),
-        SmolStr::from_iter([&name, " {…}"]),
+        match kind {
+            StructKind::Tuple => SmolStr::from_iter([&name, "(…)"]),
+            _ => SmolStr::from_iter([&name, " {…}"]),
```

Comment:
Let's write the `StructKind::Unit` arm out here ( even though we shouldn't hit it anyways I think), then we can extract this match for structs and enum-variants

Reply:
How's that look?

Reply:
👍